app-text/asciidoc/asciidoc-8.6.9-r5.ebuild

```diff
@@ -3,7 +3,7 @@
 
 EAPI=6
 
-PYTHON_COMPAT=( python2_7 )
+PYTHON_COMPAT=( python2_7 pypy )
```

Comment:
How did you determine that it didn't work?  The asciidoc test suite always passed when I tested with pypy.  Anyway, if you can wait, I'll test it tomorrow at the latest (by emerging packages that require asciidoc, that is, not just the test suite).

(Also, wasn't it @SoapGentoo who removed it about a week ago?  I was actually going to open a PR to add it back myself.)